src/general/NewGameSettings.cs

```diff
@@ -1042,6 +1048,11 @@ private void OnRandomisedGameSeedPressed()
         SetSeed(seed);
     }
 
+    private void OnWorldSizeSelected(int index)
+    {
+        worldSizeButton.Selected = index;
```

Comment:
Why does this option button select callback just apply the index back to the control this originates from?

Reply:
I've removed this the same way the other callbacks that don't actually need to do anything.